### PR TITLE
[DCJ-196] Refactor ResearchProposalVoteSlab.js to JSX

### PR DESCRIPTION
**Pull request by PintoGideon**
### Addresses

https://broadworkbench.atlassian.net/browse/DCJ-196

### Summary

1. Remove the use of react-hyperscript-helpers
2. Update the file extension from js to jsx
3. Update the tests for ResearchProposalVoteSlab component

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
Tested locally, lgtm 👍🏽

**Review by aarohinadkarni — APPROVED** [review]
tested locally looks good thank you! 👍

File: src/components/collection_voting_slab/ResearchProposalVoteSlab.jsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    </div>
+  ) : (
+    <div />
+  );
+};
+
+export const ChairVoteInfo = ({ dacVotes, isChair, adminPage = false }) => {
+
+  if (isChair && dacVotes.length > 0) {
+    return (
```
Comment (fboulnois) [inline src/components/collection_voting_slab/ResearchProposalVoteSlab.jsx]:
```suggestion
    return (isChair && dacVotes.length > 0) && (
```

Comment (fboulnois) [reply src/components/collection_voting_slab/ResearchProposalVoteSlab.jsx]: For consistency with our other previous `isRendered` components.

**Review by fboulnois — APPROVED** [review]
One suggestion:

**Pull request merged**